### ISSUE TITLE
'HsBench' object has no attribute 'timeout_clean'

### ISSUE THREAD

**Issue by ramkiperiy** [red-hat-storage/ocs-ci#4233]
TC failed in vmware setup

Error Message

test teardown failure

Stacktrace

def teardown():
>       hsbenchs3.cleanup()

tests/e2e/scale/noobaa/test_hsbench.py:17: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/hsbench.py:198: in cleanup
    self.delete_test_user()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.ocs.hsbench.HsBench object at 0x7fa2e6577e50>

    def delete_test_user(self):
        """
        Delete RGW test user and bucket belong to test user
    
        """
        log.info(f"Deleting RGW test user: {self.uid}")
        self.toolbox.exec_cmd_on_pod(
            f"radosgw-admin user rm --uid={self.uid} --purge-data",
>           timeout=self.timeout_clean,
        )
E       AttributeError: 'HsBench' object has no attribute 'timeout_clean'

ocs_ci/ocs/hsbench.py:185: AttributeError

Test run: https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/638/testReport/junit/tests.e2e.scale.noobaa.test_hsbench/TestHsBench/test_s3_benchmark_hsbench/

Complete run: https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/638/

**Comment by tiffanyn108** [comment]
Deployment of hsbench pod has issue, it doesn't deploy properly and test case didn't execute with following of clean up failed due to no attribute.  Deployment config error is seen as:
``` Events:
  Type     Reason                    Age                  From                     Message
  ----     ------                    ----                 ----                     -------
  Warning  FailedCreatePodContainer  3s (x21 over 4m21s)  kubelet                  unable to ensure pod container exists: failed to create container for [kubepods podd1cb973e-1959-4e34-bd6d-d6f380993d30] 
```
Removing node_name assignment to specific node. It allows to schedule the pod on resource availability node.

**Comment by tiffanyn108** [comment]
Fixed is merged in #4240.  Closing this PR.
